docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Routing Access Requests to Access Plugins
+description: How to set up Teleport's Access Monitoring Rules
+---
+
+With Teleport's Access Monitoring Rules access request notifications
+can be routed to plugins based on based on several conditions.
```

Comment:
I think it would be good to include one or two examples here. Also, when would a user want to use Access Monitoring Rules?